src/ray/protobuf/node_manager.proto

```diff
@@ -66,6 +66,8 @@ message RequestWorkerLeaseReply {
     // Scheduling is cancelled because task/actor is intentionally cancelled. E.g.,
     // ray.kill or ray.cancel
     SCHEDULING_CANCELLED_INTENDED = 4;
+    // Scheduling is cancelled because the task/actor is no longer schedulable.
+    SCHEDULING_CANCELLED_UNSCHEDULABLE = 5;
```

Comment:
```suggestion
    SCHEDULING_CANCELLED_NODE_NOT_AVAILABLE = 5;
```

Something like this since the node isn't available is the only cause here?

Reply:
It can also be the case where the node is alive but is infeasible. If soft is False, the task is still unschedulable in this case.